keep-only-warm-data/README.md

```diff
@@ -0,0 +1,125 @@
+# Keep only warm data into Kuzzle
+
+## Requirements
+
+Kuzzle : `>= 1.2.11`  
+Elasticsearch : `>= 5.4.1`  
+Redis : `>= 3.2`  
+
+## Introduction
+
+Kuzzle is capable of managing a very large number of documents and offering very good performances.  
+However in a Big Data context, it can be interesting to rationalize the cost of the infrastructure necessary to Kuzzle by keeping on Kuzzle only the relevant data.  
+Coupled with a secondary database, Kuzzle can be used in a Hot/Warm architecture where the most recent data is kept on Kuzzle and the oldest data in a secondary DBMS.  
+
+In most cases, deleting your oldest data from Kuzzle will only make sense if you have previously dumped it into another database. (see [How-To Synchronize Kuzzle with another database](../sync-data-to-another-database))
+
+## Architecture
+
+We will be using the standard Kuzzle stack (Kuzzle, Elasticsearch and Redis). (Check [docker-compose.yml](docker-compose.yml) for more details)
+
+On Kuzzle, the data will be stored in the `yellow-taxi` collection of the `nyc-open-data` index according to the following mapping:
+
+```js
+{
+  "pickup_datetime":  { "type": "date", "format": "MM/dd/yyyy hh:mm:ss a" },
+  "dropoff_datetime": { "type": "date", "format": "MM/dd/yyyy hh:mm:ss a" },
+  "passenger_count":  { "type": "long" },
+  "trip_distance":    { "type": "double" },
+  "pickup_position":  { "type": "geo_point" },
+  "dropoff_position": { "type": "geo_point" },
+  "fare_amount":      { "type": "double" }
+}
+```
+
+In addition to document content, Kuzzle stores a set of [metadata](https://docs.kuzzle.io/guide/essentials/document-metadata/) in Elasticsearch. These metadata are contained in the `_kuzzle_info` field (exposed as `_meta` in Kuzzle API).  
```

Comment:
The `_kuzzle_info` property is a Kuzzle implementation detail, I would mention only the `_meta` field name

Reply:
Ok. Lets not change that part for now.
I have added a new issue to our backlog though: https://github.com/kuzzleio/kuzzle-backlog/issues/257